lib/share_notify/notification_query_service.rb

```diff
@@ -0,0 +1,38 @@
+require 'share_notify/exceptions'
```

Comment:
I removed the autoload for exceptions; The exceptions classes are scoped to ShareNotify::Exceptions, but instead defined in the file.